CUE_Davis_Soil_report
================

# Year 1 (2024)

<details>
<summary>
click to open
</summary>

## Soil description

<details>
<summary>
click to open
</summary>

##### Soil samples were collected from Russell Ranch in early 2024. Trenched plots were established and weeded to minimize the impact of roots and plant communities on soil properties. Despite being fallowed, conventional plots were still treated with herbicides, leading to differences in plant abundance and community composition across fields.

##### The soils were sieved through an 8 mm mesh and stored at -80°C until aggregate fractionation. Each sample underwent two rounds of fractionation. In the first round, soils were sieved, and the resulting aggregates were dried to assess physical properties. Dried aggregates were archived for potential future use. The second round involved sieving without drying, processing only two samples at a time to maintain soil moisture. Soil moisture was adjusted to approximately 68% water holding capacity, determined for each aggregate class from the first sieve.

##### Following this, subsamples were processed as follows: some soils were stored in 1.7 mL tubes at -80°C for extra DNA extractions and metabolomics, 2 g of soil was incubated with ^18O-labeled H₂O to assess carbon use efficiency (CUE), and two 5 g subsamples were used to determine microbial biomass carbon (C) and nitrogen (N) via chloroform fumigation. After the CUE incubation, gas samples were taken from the headspace to measure respiration rates. DNA was then extracted from the soils to analyze ^18O incorporation into DNA during the 24-hour incubation. About 1 microgram of DNA was sent for metagenome analysis, and freeze dried soils were sent in for metabolomic analysis. FDA (cuvets) and POXC (microplate) were also measured using colormetric assays. Enzyme activities were measured via MUF and MC fluorometric microplate assays. OM fractions were also seperated into fPOM/oPOM/MAOM and will be sent for 13C analysis.

</details>

##### Bulk density and water infiltration of field soil n=6

<details>
<summary>
click to open
</summary>

##### Bulk density did significantly varry between conventional and organic/native. However the bulk density of organic and Native were similar. Organic had a much higher average, but the data was less even compared to Native prairie soils.

<img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-1-1.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-1-2.png" width="100%" />

</details>

## Aggregate fractionation experiment

<details>
<summary>
click to open
</summary>

##### Proportion of each aggregate size by mass n=9

<details>
<summary>
click to open
</summary>

##### In general the organic soils have higher levels of aggregation for almost all sizes except 2-0.25 mm. Distribution of aggregates by mass is more even in organic soils. These proportions do not match previous data: methods or time/space?

<img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-2-1.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-2-2.png" width="100%" />
</details>

##### All plots (No stats)

<details>
<summary>
click to open
</summary>
<img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-1.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-2.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-3.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-4.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-5.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-6.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-7.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-8.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-9.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-10.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-11.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-12.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-13.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-14.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-15.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-16.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-17.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-18.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-19.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-20.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-21.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-22.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-23.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-24.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-25.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-26.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-27.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-28.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-29.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-30.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-31.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-32.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-33.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-34.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-35.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-36.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-37.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-38.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-39.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-40.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-41.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-42.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-43.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-44.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-45.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-3-46.png" width="100%" />
</details>

##### Head space measurments (24hr CUE incubation)

<details>
<summary>
click to open
</summary>

<img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-4-1.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-4-2.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-4-3.png" width="100%" />

</details>

##### Soil analysis

<details>
<summary>
click to open
</summary>

<img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-5-1.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-5-2.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-5-3.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-5-4.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-5-5.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-5-6.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-5-7.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-5-8.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-5-9.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-5-10.png" width="100%" />

</details>

##### Biomass (Chloroform fumigation method)

<details>
<summary>
click to open
</summary>

<img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-6-1.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-6-2.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-6-3.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-6-4.png" width="100%" />

</details>

##### Enzymes activity (MUF and MC fluorimetic microplate assays)

<details>
<summary>
click to open
</summary>

<img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-7-1.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-7-2.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-7-3.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-7-4.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-7-5.png" width="100%" />

</details>

##### Fluorescein Diacetate Hydrolysis (FDA)

<details>
<summary>
click to open
</summary>

<img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-8-1.png" width="100%" />

</details>

##### Permanganate-oxidizable carbon (POXC)

<details>
<summary>
click to open
</summary>

<img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-9-1.png" width="100%" />

</details>

##### Carbon use efficency (CUE)

<details>
<summary>
click to open
</summary>

<img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-10-1.png" width="100%" />

</details>

##### Mineral associated organic matter (MAOM)

<details>
<summary>
click to open
</summary>

<img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-11-1.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-11-2.png" width="100%" />

</details>

##### Metagenome analysis

<details>
<summary>
click to open
</summary>

##### Overall heatmap

<details>
<summary>
click to open
</summary>
<img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-12-1.png" width="100%" />
</details>

##### Volcano plots show small differences in Site (Organic vs Conventional), and when looking at individual aggregates the 0.053-0.25 mm shows large differences compared with all other aggregate sizes.

<details>
<summary>
click to open
</summary>
<img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-13-1.png" width="100%" />
</details>

##### Two types of analysis showing what genes are significantly important to the comparison. Site is comparing the conventional and organicly managed plots.Note all aggregates are being compared to the Bulk soil (pre fractionated).

<details>
<summary>
click to open
</summary>

<img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-14-1.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-14-2.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-14-3.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-14-4.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-14-5.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-14-6.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-14-7.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-14-8.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-14-9.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-14-10.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-14-11.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-14-12.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-14-13.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-14-14.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-14-15.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-14-16.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-14-17.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-14-18.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-14-19.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-14-20.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-14-21.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-14-22.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-14-23.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-14-24.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-14-25.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-14-26.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-14-27.png" width="100%" />

</details>

##### Dendrogram with heat map— too small to read

<details>
<summary>
click to open
</summary>

<img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-15-1.png" width="100%" />

</details>
</details>

##### Metab analysis

<details>
<summary>
click to open
</summary>

##### Overall heatmap

<details>
<summary>
click to open
</summary>
<img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-16-1.png" width="100%" />
</details>

##### Volcano plots show small differences in Site (Organic vs Conventional), and when looking at individual aggregates the 0.053-0.25 mm shows large differences compared with all other aggregate sizes.

<details>
<summary>
click to open
</summary>
<img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-17-1.png" width="100%" />
</details>

##### Two types of analysis showing what genes are significantly important to the comparison. Site is comparing the conventional and organicly managed plots.Note all aggregates are being compared to the Bulk soil (pre fractionated).

<details>
<summary>
click to open
</summary>

<img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-18-1.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-18-2.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-18-3.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-18-4.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-18-5.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-18-6.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-18-7.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-18-8.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-18-9.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-18-10.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-18-11.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-18-12.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-18-13.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-18-14.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-18-15.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-18-16.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-18-17.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-18-18.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-18-19.png" width="100%" /><img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-18-20.png" width="100%" />

</details>

##### Dendrogram with heat map— too small to read

<details>
<summary>
click to open
</summary>

<img src="CUE_Davis_Soil_report_files/figure-gfm/unnamed-chunk-19-1.png" width="100%" />

</details>
</details>
</details>
</details>

# Year 2 (2025)

<details>
<summary>
click to open
</summary>

## Soil description

<details>
<summary>
click to open
</summary>

##### 

##### The soils were sieved through an 8 mm mesh and stored at -80°C until aggregate fractionation. Each sample underwent two rounds of fractionation. In the first round, soils were sieved, and the resulting aggregates were dried to assess physical properties. Dried aggregates were archived for potential future use. The second round involved sieving without drying, processing only two samples at a time to maintain soil moisture. Soil moisture was adjusted to approximately 68% water holding capacity, determined for each aggregate class from the first sieve.

##### Following this, subsamples were processed as follows: some soils were stored in 1.7 mL tubes at -80°C for extra DNA extractions and metabolomics, 2 g of soil was incubated with ^18O-labeled H₂O to assess carbon use efficiency (CUE), and two 5 g subsamples were used to determine microbial biomass carbon (C) and nitrogen (N) via chloroform fumigation. After the CUE incubation, gas samples were taken from the headspace to measure respiration rates. DNA was then extracted from the soils to analyze ^18O incorporation into DNA during the 24-hour incubation. About 1 microgram of DNA was sent for metagenome analysis, bulk aggregates were sent for soil property assesment, and freeze dried soils were sent in for metabolomic analysis. FDA (cuvets) and POXC (microplate) were also measured using colormetric assays. Enzyme activities were measured via MUF and MC fluorometric microplate assays. OM fractions were also seperated into fPOM/oPOM/MAOM and will be sent for 13C and 15N analysis.

</details>
</details>

# Session Info

<details>
<summary>
Session Info
</summary>

Date run: 2025-11-25

    ## R version 4.3.2 (2023-10-31 ucrt)
    ## Platform: x86_64-w64-mingw32/x64 (64-bit)
    ## Running under: Windows 11 x64 (build 26200)
    ## 
    ## Matrix products: default
    ## 
    ## 
    ## locale:
    ## [1] LC_COLLATE=English_United States.utf8 
    ## [2] LC_CTYPE=English_United States.utf8   
    ## [3] LC_MONETARY=English_United States.utf8
    ## [4] LC_NUMERIC=C                          
    ## [5] LC_TIME=English_United States.utf8    
    ## 
    ## time zone: America/Los_Angeles
    ## tzcode source: internal
    ## 
    ## attached base packages:
    ## [1] grid      stats     graphics  grDevices utils     datasets  methods  
    ## [8] base     
    ## 
    ## other attached packages:
    ##  [1] ggdendro_0.2.0         enrichplot_1.22.0      clusterProfiler_4.10.1
    ##  [4] pathview_1.42.0        edgeR_4.0.16           limma_3.58.1          
    ##  [7] rstatix_0.7.2          ropls_1.34.0           trelliscopejs_0.2.6   
    ## [10] pmartR_2.4.1           agricolae_1.3-7        knitr_1.45            
    ## [13] nlme_3.1-163           cowplot_1.1.1          ggpubr_0.6.0          
    ## [16] janitor_2.2.0          pracma_2.4.4           reshape2_1.4.4        
    ## [19] ggbiplot_0.55          scales_1.3.0           vegan_2.6-4           
    ## [22] lattice_0.21-9         permute_0.9-7          lubridate_1.9.3       
    ## [25] forcats_1.0.0          stringr_1.5.1          purrr_1.0.2           
    ## [28] readr_2.1.4            tidyr_1.3.0            tibble_3.2.1          
    ## [31] ggplot2_3.5.2          tidyverse_2.0.0        dplyr_1.1.4           
    ## [34] plyr_1.8.9             tarchetypes_0.11.0     targets_1.9.1         
    ## 
    ## loaded via a namespace (and not attached):
    ##   [1] splines_4.3.2               ggplotify_0.1.2            
    ##   [3] bitops_1.0-7                polyclip_1.10-7            
    ##   [5] graph_1.80.0                XML_3.99-0.18              
    ##   [7] lifecycle_1.0.4             processx_3.8.2             
    ##   [9] MASS_7.3-60                 MultiAssayExperiment_1.28.0
    ##  [11] backports_1.4.1             magrittr_2.0.3             
    ##  [13] rmarkdown_2.29              yaml_2.3.7                 
    ##  [15] DBI_1.2.3                   RColorBrewer_1.1-3         
    ##  [17] abind_1.4-8                 zlibbioc_1.48.0            
    ##  [19] GenomicRanges_1.54.1        ggraph_2.2.1               
    ##  [21] BiocGenerics_0.48.1         RCurl_1.98-1.14            
    ##  [23] yulab.utils_0.1.8           tweenr_2.0.3               
    ##  [25] GenomeInfoDbData_1.2.11     IRanges_2.36.0             
    ##  [27] S4Vectors_0.40.2            ggrepel_0.9.6              
    ##  [29] tidytree_0.4.6              codetools_0.2-19           
    ##  [31] DelayedArray_0.28.0         DOSE_3.28.2                
    ##  [33] ggforce_0.4.2               tidyselect_1.2.1           
    ##  [35] aplot_0.2.4                 farver_2.1.1               
    ##  [37] viridis_0.6.5               matrixStats_1.2.0          
    ##  [39] stats4_4.3.2                base64enc_0.1-3            
    ##  [41] webshot_0.5.5               jsonlite_1.8.8             
    ##  [43] tidygraph_1.3.1             ggridges_0.5.6             
    ##  [45] iterators_1.0.14            systemfonts_1.0.5          
    ##  [47] foreach_1.5.2               ggnewscale_0.5.1           
    ##  [49] tools_4.3.2                 progress_1.2.3             
    ##  [51] ragg_1.2.6                  treeio_1.26.0              
    ##  [53] snow_0.4-4                  Rcpp_1.0.11                
    ##  [55] glue_1.6.2                  gridExtra_2.3              
    ##  [57] SparseArray_1.2.3           xfun_0.52                  
    ##  [59] mgcv_1.9-0                  qvalue_2.34.0              
    ##  [61] MatrixGenerics_1.14.0       GenomeInfoDb_1.38.8        
    ##  [63] withr_3.0.2                 fastmap_1.2.0              
    ##  [65] AlgDesign_1.2.1             callr_3.7.6                
    ##  [67] digest_0.6.33               gridGraphics_0.5-1         
    ##  [69] secretbase_1.0.3            timechange_0.2.0           
    ##  [71] R6_2.5.1                    textshaping_0.3.7          
    ##  [73] colorspace_2.1-0            GO.db_3.18.0               
    ##  [75] RSQLite_2.3.9               utf8_1.2.4                 
    ##  [77] generics_0.1.3              calibrate_1.7.7            
    ##  [79] data.table_1.14.8           prettyunits_1.2.0          
    ##  [81] graphlayouts_1.2.2          httr_1.4.7                 
    ##  [83] S4Arrays_1.2.0              scatterpie_0.2.4           
    ##  [85] pkgconfig_2.0.3             gtable_0.3.6               
    ##  [87] blob_1.2.4                  XVector_0.42.0             
    ##  [89] shadowtext_0.1.4            htmltools_0.5.8.1          
    ##  [91] carData_3.0-5               fgsea_1.28.0               
    ##  [93] base64url_1.4               autocogs_0.1.4             
    ##  [95] ggupset_0.4.1               Biobase_2.62.0             
    ##  [97] png_0.1-8                   snakecase_0.11.1           
    ##  [99] ggfun_0.1.8                 rstudioapi_0.17.1          
    ## [101] tzdb_0.4.0                  checkmate_2.3.0            
    ## [103] org.Hs.eg.db_3.18.0         cachem_1.1.0               
    ## [105] parallel_4.3.2              HDO.db_0.99.1              
    ## [107] AnnotationDbi_1.64.1        pillar_1.10.0              
    ## [109] vctrs_0.6.4                 car_3.1-2                  
    ## [111] cluster_2.1.4               evaluate_1.0.1             
    ## [113] Rgraphviz_2.46.0            KEGGgraph_1.62.0           
    ## [115] qqman_0.1.9                 cli_3.6.1                  
    ## [117] locfit_1.5-9.12             compiler_4.3.2             
    ## [119] rlang_1.1.2                 crayon_1.5.3               
    ## [121] ggsignif_0.6.4              MultiDataSet_1.30.0        
    ## [123] labeling_0.4.3              mclust_6.0.1               
    ## [125] ps_1.8.1                    fs_1.6.5                   
    ## [127] stringi_1.8.2               viridisLite_0.4.2          
    ## [129] BiocParallel_1.36.0         munsell_0.5.1              
    ## [131] Biostrings_2.70.3           lazyeval_0.2.2             
    ## [133] GOSemSim_2.28.1             Matrix_1.6-1.1             
    ## [135] patchwork_1.3.0             hms_1.1.3                  
    ## [137] bit64_4.5.2                 KEGGREST_1.42.0            
    ## [139] statmod_1.5.0               highr_0.11                 
    ## [141] SummarizedExperiment_1.32.0 igraph_2.1.2               
    ## [143] broom_1.0.7                 memoise_2.0.1              
    ## [145] ggtree_3.10.1               fastmatch_1.1-6            
    ## [147] bit_4.5.0.1                 DistributionUtils_0.6-1    
    ## [149] gson_0.1.0                  ape_5.7-1

</details>
